docs/source/overview/options.md

```diff
@@ -141,19 +141,23 @@ information of the evaluator contains following items:
 
     - `type: [str]` The type of the metric. The supported types are `accuracy`, `latency` and `custom`.
 
-    - `subtype: [str]` The subtype of the metric. Please refer to [AccuracySubtype](accuracy_sub_type) and
-    [LatencySubtype](latency_sub_type) for the supported sub-types. It is `null` for `custom` type.
+    - `subtyps: [List[Dict]]` The subtypes of the metric. Cannot be null or empty. Each subtype is a dictionary that contains following items:
 
-    - `higher_is_better: [Boolean]` True if the metric is better when it is higher. It is `true` for `accuracy` type and `false` for `latency` type.
+        - `name: str` The name of the subtype. Please refer to [AccuracySubtype](accuracy_sub_type) and [LatencySubtype](latency_sub_type)
```

Comment:
I think we need a name for the custom metrics too? It cannot be null?

Reply:
Updated.